libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -49,8 +56,6 @@ class Config {
         // versions
         releaseVersion = project.versions.release
         nextVersion = project.versions.development
-
-        resolveProperties()
     }
 
     public printConfig = { ->
```

Comment:
Can we use `void printConfig()` and `void resolveProperties()` instead of the "lambda" style?